packages/db/README.md

```
@@ -2,11 +2,47 @@
 
 [![Apache-2.0 licensed][badge-license]][url-license]
 
+Database connection utilities for Esposter — server environment only. Provides composables for Drizzle ORM (PostgreSQL), Azure Table Storage, Azure Blob Storage, Azure AI Search, and Azure WebPubSub.
+
 ### Table of Contents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)